extensions/dacpac/src/utils.ts

```diff
@@ -22,14 +23,21 @@ export function getPackageInfo(packageJson: any): IPackageInfo | undefined {
 }
 
 /**
- * Map an error message into a short name for the type of error.
- * @param msg The error message to map
+ * Get file size from the file stats using the file path uri
+ * @param uri The file path
  */
-export function getTelemetryErrorType(msg: string): string {
-	if (msg && msg.indexOf('Object reference not set to an instance of an object') !== -1) {
-		return 'ObjectReferenceNotSet';
+export async function getFileSize(uri: string): Promise<number | undefined> {
```

Comment:
I'd follow the c# convention here and name these something like `tryGetFileSize` or `safeGetFileSize` to indicate that we won't throw if an error occurs.

Reply:
(same for the other function as well. Although honestly I'd just combine them into a single function since you aren't reusing the private one anywhere else and are unlikely to in the future)